packages/@local/package-chores/scripts/smoke-test-create-block-app.ts

```diff
@@ -36,7 +37,7 @@ const defaultExecaOptions = {
 } as const;
 
 const script = async () => {
-  console.log(chalk.bold("E2E-testing create-block-app..."));
+  console.log(chalk.bold("Smoke-testing create-block-app..."));
```

Comment:
This should have been a part of https://github.com/blockprotocol/blockprotocol/pull/645, but I've missed this case.